files/etc/slurm-llnl/slurm.conf

```diff
@@ -164,26 +165,69 @@ SlurmSchedLogFile=/var/log/slurm-llnl/sched.log
-{% endfor %}
-{% for node, values in pillar['slurm']['nodes']['interactive']['cpus'].items() %}
-NodeName={{ node }} RealMemory={{ values.mem }} CPUs={{ values.cores }} State=UNKNOWN
-{% endfor %}
-{% for node, values in pillar['slurm']['nodes']['batch']['gpus'].items() %}
-NodeName={{ node }} RealMemory={{ values.mem }} CPUs={{ values.cores }} Gres={{ (values.gres.items()|join(',')).replace('(','').replace(')','').replace("', ",':').replace("'",'') }} State=UNKNOWN
-{% endfor %}
-# Partitions
-PartitionName=long Nodes={{ ','.join(pillar['slurm']['nodes']['batch']['cpus']) }} Default=YES MaxTime=43200
-PartitionName=short Nodes={{ ','.join(pillar['slurm']['nodes']['batch']['cpus']) }} Default=NO MaxTime=60 Priority=5000
-PartitionName=gpus  Nodes={{ ','.join(pillar['slurm']['nodes']['batch']['gpus']) }} Default=NO MaxTime=10080
-PartitionName=interactive Nodes={{ ','.join(pillar['slurm']['nodes']['interactive']['cpus']) }} Default=NO MaxTime=4320 Priority=7000 PreemptMode=OFF
 
-{% set rocsList = [] %}
-{% for node, values in pillar['slurm']['nodes']['batch']['cpus'].items() %} {% if node.startswith('roc') %} {% set rocsListTrash = rocsList.append(node) %} {% endif %} {% endfor %}
+NodeName=biomedia01 RealMemory=64000 CPUs=24 State=UNKNOWN
+
+NodeName=biomedia02 RealMemory=63800 CPUs=24 State=UNKNOWN
+
+NodeName=biomedia05 RealMemory=64414 CPUs=24 State=UNKNOWN
+
+NodeName=biomedia06 RealMemory=128850 CPUs=64 State=UNKNOWN
+
+NodeName=biomedia07 RealMemory=128850 CPUs=64 State=UNKNOWN
+
+NodeName=biomedia08 RealMemory=128850 CPUs=64 State=UNKNOWN
+
+NodeName=biomedia09 RealMemory=128850 CPUs=64 State=UNKNOWN
+
+NodeName=biomedia10 RealMemory=128851 CPUs=24 State=UNKNOWN
+
+NodeName=biomedia11 RealMemory=252000 CPUs=32 State=UNKNOWN
+
+NodeName=roc01 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc02 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc03 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc04 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc05 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc06 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc07 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc08 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc09 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc10 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc11 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc12 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc13 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc14 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc15 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=roc16 RealMemory=257869 CPUs=32 State=UNKNOWN
+
+NodeName=monal01 RealMemory=80000 CPUs=12 Gres=gpu:8 State=UNKNOWN
+
+# Partitions
+PartitionName=long Nodes=biomedia01,biomedia02,biomedia05,biomedia06,biomedia07,biomedia08,biomedia09,biomedia10,roc01,roc02,roc03 Default=YES MaxTime=43200
```

Comment:
all roc machines should be in long as well. it's fine to have two partitions overlapping